agent/trufflehog_agent.py

```diff
@@ -87,7 +87,8 @@ def process(self, message: m.Message) -> None:
             content = response.get("body", b"") + b"\n" + request.get("body", b"")
             cmd_output = process_input.process_file(content)
         else:
-            raise ValueError(f"Unsupported selector {message.selector}.")
+            logging.error(f"Unsupported selector {message.selector}.")
```

Comment:
Not the correct fix

Reply:
Yes I created another PR with the right fix, https://github.com/Ostorlab/agent_trufflehog/pull/7.